evm_vs_ewasm_opcode_table.md

```diff
@@ -0,0 +1,120 @@
+
+EVM opcode            |     ewasm interface method
+----------------------|-----------------------------
+0x30: ADDRESS         |      getAddress
+0x31: BALANCE         |      getBalance
+0x32: ORIGIN          |      getTxOrigin
+0x33: CALLER          |      getCaller
+0x34: CALLVALUE       |      getCallValue
+0x35: CALLDATALOAD    |      callDataCopy(resultOffset, dataOffset, 32)
+0x36: CALLDATASIZE    |      getCallDataSize
+0x37: CALLDATACOPY    |      callDataCopy(resultOffset, dataOffset, size)
+0x38: CODESIZE        |      getCodeSize
+0x39: CODECOPY        |      codeCopy
+0x3a: GASPRICE        |      getTxGasPrice
+0x3b: EXTCODESIZE     |      getExternalCodeSize
+0x3c: EXTCODECOPY     |      externalCodeCopy
+0x3d: RETURNDATASIZ   |      getReturnDataSize
+0x3e: RETURNDATACOP   |      returnDataCopy
+0x40: BLOCKHASH       |      getBlockHash
+0x41: COINBASE        |      getBlockCoinbase
+0x42: TIMESTAMP       |      getBlockTimestamp
+0x43: NUMBER          |      getBlockNumber
+0x44: DIFFICULTY      |      getBlockDifficulty
+0x45: GASLIMIT        |      getBlockGasLimit
+0x54: SLOAD           |      storageLoad
+0x55: SSTORE          |      storageStore
+0x58: PC              |      -
+0x5a: GAS             |      getGasLeft
+0xa0: LOG0            |      log(0,..)
+...                   |
+0xa4: LOG4            |      log(4,..)
+0xf0: CREATE          |      create
+0xf1: CALL            |      call
+0xf2: CALLCODE        |      callCode
+0xf3: RETURN          |      return
+0xf4: DELEGATECALL    |      callDelegate
+0xfa: STATICCALL      |      callStatic
+0xfd: REVERT          |      revert
+0xff: SELFDESTRUCT    |      selfDestruct
+
+
+
+## EVM opcodes vs WebAssembly instructions
+
+Whereas EVM is an untyped VM with a 256-bit word size, WebAssembly is a typed VM with 32-bit and 64-bit word sizes. The two wasm base types are integers and floats, which when combined with the two word sizes, create four types: i32, i64, f32, f64.
+
+Ethereum WebAssembly, or ewasm, is a subset of wasm. Only the integer types (i32 and i64) are supported, floating point types and floating point instructions are not supported. In total, 60 wasm instructions are supported in ewasm:
+* 10 control flow instructions
+* 11 basic instructions
+* 19 integer arithmetic instructions
+* 10 integer comparison instructions
+* 3 integer conversion instructions
+* 7 memory instructions (load, store, extend)
+
+ewasm excludes the wasm floating point instructions:
+* 14 floating point arithmetic instructions
+* 6 floating point comparison instructions
+* 7 floating point conversion instructions
+
+To reference wasm instructions, see the [wasm spec](https://github.com/WebAssembly/design/blob/master/Semantics.md) or [reference manual](https://github.com/sunfishcode/wasm-reference-manual/blob/master/WebAssembly.md#instructions).
+
+
+### EVM opcodes that translate to wasm instructions
```

Comment:
This is misleading. EVM opcodes are 256bit wide, they do not translate to wasm instructions.

Reply:
How about "EVM opcodes that reduce to wasm instructions"

Reply:
Actually I think the wording depends on the goal of the document. Does the document tries to be a quick onboarding to ewasm for those who are familiar with EVM or does it try to explain how a potential EVM to wasm translator could work?